fs/dtype_test.go

```diff
@@ -0,0 +1,18 @@
+package fs
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd/testutil"
+)
+
+func TestRequiresRootNOP(t *testing.T) {
+
+	// This is a dummy test case that exist to call
```

Comment:
Oh I see. I don't think that's still the case now, the `test.root` flag is defined regardless of the platform in `testutils/helpers.go`.